SCF-Guess-Assembly-Line.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+import numpy as np
+
+def SDF_to_coords(filename, num_atom):
+    directory = r'/mnt/c/Users/jacob/OneDrive/Documents/Structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
```

Comment:
For some reason there seems to be two versions of the same directory in windows and linux (I have enabled the windows subsystem for linux). In a linux terminal I moved the SDF file folder ("SDFs") into scf_guess/, which created scf_guess/SDFs. But it exists in windows as what you see on line 6. This being said, it doesn't seem to be causing problems. What do you think?

Anyway I get an error where this comment is, line 9. To test with the new directory, I entered SDF_to_coords("Guanidine", 9) in a cell that is separate from the entire code.  It tried to retrieve "Aletamine", and said that it did not exist (it does). Looking at the code, there's no apparent reason why it should be doing this.

Reply:
Thanks Ryan, I just read your email.

Reply:
Filesystems are complicated beasts. Windows and Linux do things very differently so I can only guess at what Windows is doing under the hood to try to provide a unified API. I sent you an email earlier today about the `guess_driver` function I put together, in theory it should alleviate most of these problems.